### ISSUE TITLE
latest release uninstallable with ruby 3.0

### ISSUE THREAD

**Issue by terceiro** [diaspora/diaspora_federation#123]
```
$ ruby -v
ruby 3.0.2p107 (2021-07-07 revision 0db68f0233) [x86_64-linux-gnu]
$ cat Gemfile 
───────┬─────────────────────────────────────────────
       │ File: Gemfile
───────┼─────────────────────────────────────────────
   1   │ source "https://rubygems.org"
   2   │ gem "diaspora_federation-rails", "0.2.7"
───────┴─────────────────────────────────────────────
$ bundle 
Fetching gem metadata from https://rubygems.org/............
Resolving dependencies...
Bundler found conflicting requirements for the Ruby version:
  In Gemfile:
    Ruby

    diaspora_federation-rails (= 0.2.7) was resolved to 0.2.7, which depends on
      Ruby (~> 2.1)
```

I see that this requirement was relaxed in commit a1613577e874706691bdc454e10df7229fee8bfd, but it seems that didn't get into a release yet?

**Comment by terceiro** [comment]
weirdly, if I drop the version spec from the `Gemfile`, it resolves to 0.0.13?

```
$ cat Gemfile 
───────┬──────────────────────────────────────────────────────────────
       │ File: Gemfile
───────┼──────────────────────────────────────────────────────────────
   1   │ source "https://rubygems.org"
   2   │ gem "diaspora_federation-rails"
───────┴──────────────────────────────────────────────────────────────
$ bundle 
Fetching gem metadata from https://rubygems.org/............
Resolving dependencies...
Bundler found conflicting requirements for the Ruby version:
  In Gemfile:
    Ruby

    diaspora_federation-rails was resolved to 0.0.13, which depends on
      Ruby (~> 2.0)
```

**Comment by SuperTux88** [comment]
Version 0.3.0 supports ruby 3.0 and 3.1 now.
